metabase/metabase
# Remove time-series grouping widget for custom date breakout

Closes https://github.com/metabase/metabase/issues/33504

### Description

As Alex P. has pointed out in [this comment](https://github.com/metabase/metabase/issues/33504#issuecomment-1695571921), we shouldn't display the time-series grouping widget for custom date breakout because we don't support it.

This PR removes the time-series grouping widget when we summarize by custom date field. And ensure we won't display mode footer when the mode is outdated. e.g. when modifying a question without rerunning the result. Since `mode` relies on the last run result.

### How to verify

1. New > Question > Orders
1. Add a custom column `[Created At]` name it `Custom Created At`
1. Summarize count by `Custom Created At`
1. Click **Visualize**

You shouldn't see the time-series grouping widget in the footer.

### Demo

#### After
![Screenshot 2023-09-07 at 11 29 40 PM](https://github.com/metabase/metabase/assets/1937582/f9ebca65-fdfc-4404-9eb2-864289330477)

#### Before
![Screenshot 2023-09-07 at 11 30 01 PM](https://github.com/metabase/metabase/assets/1937582/dfa2f0cf-6e2e-46f8-bf8e-7ae7a10d866c)


### Checklist
